packages/dev/s2-docs/src/VisualExampleClient.tsx

```diff
@@ -1018,8 +1018,8 @@ function PlacementControl({control, value, onChange}) {
             "sb .  .  .  eb"
             ".  bs bc be . "
           `,
-          gridTemplateColumns: '25px 24px 24px 25px 24px',
-          gridTemplateRows: '25px 24px 24px 25px 24px'
+          gridTemplateColumns: 'calc(25px * var(--s2-scale)) calc(24px * var(--s2-scale)) calc(24px * var(--s2-scale)) calc(25px * var(--s2-scale)) calc(24px * var(--s2-scale))',
+          gridTemplateRows: 'calc(25px * var(--s2-scale)) calc(24px * var(--s2-scale)) calc(24px * var(--s2-scale)) calc(25px * var(--s2-scale)) calc(24px * var(--s2-scale))'
```

Comment:
Can't we use an array of numbers here and the style macro will do this automatically?


```suggestion
          gridTemplateRows: [25, 24, 24, 25, 24]
```